src/main/java/com/onthegomap/planetiler/openmaptiles/util/Utils.java

```diff
@@ -1,78 +0,0 @@
-package com.onthegomap.planetiler.openmaptiles.util;
```

Comment:
we ended up not moving utils to core, let's revert this change which should fix the tests as well.

Reply:
@msbarry Reverted. It passed the tests.